scipy/optimize/_qap.py

```diff
@@ -50,21 +50,21 @@ def quadratic_assignment(A, B, method="faq", options=None):
     options : dict, optional
         A dictionary of solver options. All solvers support the following:
 
-        partial_match : 2d-array of integers, optional, (default = None)
+        maximize : bool (default: False)
+            Setting `maximize` to ``True`` solves the Graph Matching Problem
+            (GMP) rather than the Quadratic Assignment Problem (QAP).
+
+        partial_match : 2-D array of integers, optional (default: None)
             Allows the user to fix part of the matching between the two
             matrices. In the literature, a partial match is also
             known as a "seed" [2]_.
 
             Each row of `partial_match` specifies the indices of a pair of
             corresponding nodes, that is, node ``partial_match[i, 0]`` of `A`
             is matched to node ``partial_match[i, 1]`` of `B`. Accordingly,
-            ``partial_match`` is an array of size ``(m , 2)``, where ``m`` is
+            ``partial_match`` is an array of size ``(m, 2)``, where ``m`` is
             not greater than the number of nodes.
 
-        maximize : bool (default = False)
-            Setting `maximize` to ``True`` solves the Graph Matching Problem
-            (GMP) rather than the Quadratic Assingnment Problem (QAP).
-
         rng : {None, int, `~np.random.RandomState`, `~np.random.Generator`}
```

Comment:
```suggestion
        rng : {None, int, `~np.random.RandomState`, `~np.random.Generator`} (default: None)
```
Might as well specify the default in this line since we're doing that for all the other options. You could also remove it from the end of the parameter description.

Reply:
Also, since we've been at this level of nit for a while:
![image](https://user-images.githubusercontent.com/6570539/93125631-4a3d4e80-f680-11ea-98e8-37fdafafd6f1.png)
Sometimes RandomState has single backticks, sometimes double. My understanding is that options/parameters standing alone should be single backtick, but code involving them should be double. 

Not that there's any hope of consistency, given that our sphinx theme renders the parameters in bold - not italicized OR teletype - in the headings of the Parameters section. Personally I think they should _always_ be teletype.

Reply:
We were missing a default on the first line here. I couldn't fit it all in 80 chars whilst including the default. Instead, I adapted the description from [`spatial.transform.Rotation.random'](https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.transform.Rotation.random.html#scipy.spatial.transform.Rotation.random) which is very similar to our needs here and very concise.